<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAudioDeviceVolumeSetupComponent</class>
 <widget class="QFrame" name="CAudioDeviceVolumeSetupComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio setup</string>
  </property>
  <layout class="QGridLayout" name="gl_AudioDeviceVolumeSetupComponent">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="8" column="3">
    <widget class="QToolButton" name="tb_ResetInVolume">
     <property name="text">
      <string>...</string>
     </property>
      <property name="icon">
        <iconset resource="../../blackmisc/blackmisc.qrc">
          <normaloff>:/pastel/icons/pastel/16/undo.png</normaloff>:/pastel/icons/pastel/16/undo.png</iconset>
      </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lbl_VolumeIn">
     <property name="text">
      <string>In</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_Info">
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSlider" name="hs_VolumeIn">
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbl_SetupAudioOutputDevice">
     <property name="text">
      <string>Out</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="cb_DisableAudioEffects">
     <property name="text">
      <string>Disable realistic audio simulation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="le_Info">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>extra info goes here</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="wi_OutDevice" native="true">
     <layout class="QHBoxLayout" name="hl_OutDevice">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="cb_SetupAudioOutputDevice"/>
      </item>
      <item>
       <widget class="QToolButton" name="tb_RefreshOutDevice">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../blackmisc/blackmisc.qrc">
          <normaloff>:/diagona/icons/diagona/icons/arrow-circle-135.png</normaloff>:/diagona/icons/diagona/icons/arrow-circle-135.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lbl_SetupAudioLoopback">
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QToolButton" name="tb_ResetOutVolumeCom1">
     <property name="text">
      <string>...</string>
     </property>
      <property name="icon">
        <iconset resource="../../blackmisc/blackmisc.qrc">
          <normaloff>:/pastel/icons/pastel/16/undo.png</normaloff>:/pastel/icons/pastel/16/undo.png</iconset>
      </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QToolButton" name="tb_ResetOutVolumeCom2">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../blackmisc/blackmisc.qrc">
        <normaloff>:/pastel/icons/pastel/16/undo.png</normaloff>:/pastel/icons/pastel/16/undo.png</iconset>
      </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lbl_VolumeOutCom1">
     <property name="text">
      <string>Out COM1</string>
     </property>
    </widget>
   </item>
  <item row="12" column="0">
    <widget class="QLabel" name="lbl_VolumeOutCom2">
     <property name="text">
      <string>Out COM2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="fr_TxRx">
     <layout class="QHBoxLayout" name="hl_TxRx">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="BlackGui::CLedWidget" name="led_AudioConnected" native="true"/>
      </item>
      <item>
       <widget class="BlackGui::CLedWidget" name="led_Rx1" native="true"/>
      </item>
      <item>
       <widget class="BlackGui::CLedWidget" name="led_Rx2" native="true"/>
      </item>
      <item>
       <spacer name="hs_AudioEffects">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_1Rec">
        <property name="text">
         <string>Rx1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_1Tx">
        <property name="text">
         <string>Tx1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_2Rec">
        <property name="text">
         <string>Rx2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_2Tx">
        <property name="text">
         <string>Tx2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_IntegratedWithCom">
        <property name="toolTip">
         <string>integrated with sim. cockpit</string>
        </property>
        <property name="text">
         <string>sync.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="wi_InDevice" native="true">
     <layout class="QHBoxLayout" name="hl_InDevice">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="cb_SetupAudioInputDevice">
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tb_RefreshInDevice">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../blackmisc/blackmisc.qrc">
          <normaloff>:/diagona/icons/diagona/icons/arrow-circle-135.png</normaloff>:/diagona/icons/diagona/icons/arrow-circle-135.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QWidget" name="wi_TestAudio" native="true">
     <layout class="QHBoxLayout" name="hl_TestAudio">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cb_SetupAudioLoopback">
        <property name="text">
         <string>Loopback, test sound in- to output</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QSlider" name="hs_VolumeOutCom1">
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QSlider" name="hs_VolumeOutCom2">
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  <item row="13" column="0">
    <widget class="QLabel" name="lbl_OutLevelMeter">
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="BlackGui::CLevelMeter" name="wip_OutLevelMeter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  <item row="9" column="0">
    <widget class="QLabel" name="lbl_InLevelMeter">
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
  </item>
   <item row="9" column="1">
    <widget class="BlackGui::CLevelMeter" name="wip_InLevelMeter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_SetupAudioInputDevice">
     <property name="text">
      <string>In</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::CLevelMeter</class>
   <extends>QFrame</extends>
   <header>blackgui/levelmeter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::CLedWidget</class>
   <extends>QWidget</extends>
   <header>blackgui/led.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Info</tabstop>
  <tabstop>cb_1Rec</tabstop>
  <tabstop>cb_1Tx</tabstop>
  <tabstop>cb_2Rec</tabstop>
  <tabstop>cb_2Tx</tabstop>
  <tabstop>cb_IntegratedWithCom</tabstop>
  <tabstop>cb_SetupAudioInputDevice</tabstop>
  <tabstop>tb_RefreshInDevice</tabstop>
  <tabstop>cb_SetupAudioOutputDevice</tabstop>
  <tabstop>tb_RefreshOutDevice</tabstop>
  <tabstop>cb_DisableAudioEffects</tabstop>
  <tabstop>cb_SetupAudioLoopback</tabstop>
  <tabstop>hs_VolumeIn</tabstop>
  <tabstop>tb_ResetInVolume</tabstop>
  <tabstop>hs_VolumeOutCom1</tabstop>
  <tabstop>tb_ResetOutVolumeCom1</tabstop>
  <tabstop>hs_VolumeOutCom2</tabstop>
  <tabstop>tb_ResetOutVolumeCom2</tabstop>
 </tabstops>
 <resources>
  <include location="../../blackmisc/blackmisc.qrc"/>
 </resources>
 <connections/>
</ui>
